stellar-dotnet-sdk/WebAuthentication.cs

```diff
@@ -98,6 +123,12 @@ public static class WebAuthentication
             if (transaction.SequenceNumber != 0)
                 throw new InvalidWebAuthenticationException("Challenge transaction sequence number must be 0");
 
+            if (!(transaction is Transaction))
```

Comment:
it seems like this type check is not necessary because transaction is declared to be of type `Transaction`

Reply:
Correct. I missed that.